index.css

```diff
@@ -906,3 +619,42 @@ textarea[id=subject] {
   border: none;
   padding-left: 10px;
 }
+
+/* destop reponsiveness */
+@media only screen and (min-width: 768px) {
+  .contact-container {
+    flex-direction: row;
+    width: 1000px;
+  }
+
+  /* .info{
+    flex-direction: row;
+    width: 1000px;
+    display: flex;
+  } */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.